src/meta/src/hummock/manager/mod.rs

```diff
@@ -1380,36 +1404,38 @@ where
             .collect();
 
         let versioning = versioning_guard.deref_mut();
-        let (mut new_version_delta, mut new_hummock_version) =
-            match self.sync_group(versioning, &compaction_groups).await? {
-                Some((entry_k, entry_v, new_hummock_version)) => (
-                    BTreeMapEntryTransaction::new_insert(
-                        &mut versioning.hummock_version_deltas,
-                        entry_k,
-                        entry_v,
-                    ),
-                    new_hummock_version,
+        let (mut new_version_delta, mut new_hummock_version) = match self
```

Comment:
Why do we still need sync_group in commit_epoch, given that we've done that in register/unregister in CompactionGroupManager?

Reply:
> Why do we still need sync_group in commit_epoch, given that we've done that in register/unregister in CompactionGroupManager?

We can remove `sync_group` here.